tox.ini

```diff
@@ -11,38 +11,18 @@ passenv =
     #http://tox.readthedocs.io/en/latest/config.html#confval-passenv=SPACE-SEPARATED-GLOBNAMES
     *
 commands =
-    pip install -v .
-    py.test -v --timeout 60 --ignore=tmp-build --import-mode append {posargs}
+    # Install main package and all sub-packages
+    pip install . .[avro] .[schema-registry] .[json] .[protobuf]
+    # Early verification that module is loadable
+    python -c 'import confluent_kafka ; print(confluent_kafka.version())'
+    # Run tests (large timeout to allow docker image downloads)
+    python -m pytest --timeout 600 --ignore=tmp-build {posargs}
     # See tests/README.md for additional notes on testing
-    python tests/integration/integration_test.py
+    #python tests/integration/integration_test.py
 
-[base]
 deps =
     # https://docs.pytest.org/en/latest/changelog.html#id53
-    pytest==4.6.4
-    pytest-timeout
-    fastavro
-    requests
-
-[testenv:py27]
-deps =
-    {[base]deps}
-    avro
-
-[testenv:py34]
-deps =
-    {[base]deps}
-    avro-python3
-
-[testenv:py35]
-deps =
-    {[base]deps}
-    avro-python3
-
-[testenv:py36]
-deps =
-    {[base]deps}
-    avro-python3
+    -rtests/requirements.txt
```

Comment:
Missing space

Reply:
That's actually required to be without space.

Reply:
TIL